amethyst_config/rustfmt.toml

```diff
@@ -1 +1,4 @@
 max_width = 200
+reorder_imports = true
+reorder_import_names = true
+reorder_imports_in_group = true
```

Comment:
Can you also stick these in /rustfmt.toml?  There's no need for amethyst_config to have formatting different from the rest of the repository right?

Reply:
They are already, in atleast one of the other PRs.

Reply:
I can remove the config specific config file.

Reply:
That would be preferable